identity/webapp/src/frontend/components/PageWrapper.tsx

```diff
@@ -30,11 +30,11 @@ export const PageWrapper: FunctionComponent<PropsWithChildren<Props>> = ({
   const { isEnhanced } = useContext(AppContext);
   const { collectionVenues } = usePrismicData();
   const venues = transformCollectionVenues(collectionVenues);
-
+  const fullTitle = `${title} | Wellcome Collection`;
```

Comment:
We started getting this warning: 'A title element received an array with more than 1 element as children'

This ensures there is only one child text node

Reply:
😮